app/configurations/config.hbnext.js

```diff
@@ -180,7 +180,13 @@ export default configMerger(walttiConfig, {
         showLayerSelector: true, // DT-3470
         showStopMarkerPopupOnMobile: false, // DT-3470
         showScaleBar: true, // DT-3470, DT-3397
-
+        genericMarker: {
+            popup: {
+                offset: [0,0],
+                maxWidth: 250,
```

Comment:
Is maxWidth==minWidth reasonable? Or will we see truncated text then?

Reply:
It is actually not necessary, with or without them they look the same. I just thought its better if they have the same size but if you want I can remove it. The important property is the `offset` so the popup is not shifted from the icon.